selfapi.js

```diff
@@ -6,6 +6,13 @@ var https = require('https');
 var nodepath = require('path');
 var url = require('url');
 
+function ensureSerialized(strOrObject, spaces=null) {
+  if (strOrObject instanceof Object) {
+    return JSON.stringify(strOrObject, null, spaces);
+  }
+  return strOrObject;
```

Comment:
Handling `undefined` or `null` could be done I think. But I don't see an easy way of handling JSON strings. What do you think about it? Should it be fixed?